tests/modules/apigee_mock_client/src/MockHttpClientFactory.php

```diff
@@ -72,14 +72,14 @@ public function __construct(HandlerStack $stack, MockHandler $mock_stack, StateI
     $this->mock_stack = $mock_stack;
 
     // Check for the integration enabled environment variable.
-    if ($enabled = getenv('APIGEE_INTEGRATION_ENABLE')) {
+    if ($enabled = getenv('APIGEE_MOCK_CLIENT_INTEGRATION_ENABLE')) {
```

Comment:
Can we change this to `APIGEE_EDGE_INTEGRATION_ENABLED` to be more closely aligned with the other environment variables that are documented in `CONTRIBUTING.md`.